Update about page

matthieu-lapeyre opened the pull request:
suggestion for the about page

@jgrizou if you can help to make it better ...

matthieu-lapeyre committed:
modify the content of the About page

matthieu-lapeyre committed:
re-order section and add contribution info

damiencaselli commented on _pages/about/en.md:
```diff
@@ -11,50 +11,43 @@ published: true
-Our major challenge is to make the robotic hardware cheap and modular so anyone can change the morphology of his/her robot to be adapted for the experience targeted.
-
-Of course, all this work is open source and distributed under Creative Commons BY-SA licenses.
+The Poppy project is born in 2012 in the [Flowers laboratory](https://flowers.inria.fr/) at [Inria Bordeaux Sud-Ouest](http://www.inria.fr/en/centre/bordeaux).
```
_was born_

damiencaselli commented on _pages/about/en.md:
```diff
@@ -11,50 +11,43 @@ published: true
+ - Art with the need of animator to create [the illusion of life](http://en.wikipedia.org/wiki/Disney_Animation:_The_Illusion_of_Life) and emotions.
 
-Poppy creatures are being integrated in the v-rep simulator with a transparent switch from simulated to physical robots through pypot.
+If you have no idea how you could help but you would want to, you are very welcome and you can **take a look at open issues on our [GitHub](https://github.com/poppy-project/) and [call for contributions](https://forum.poppy-project.org/tags/call-for-contributions)**.
```
We should link to issues here, not repo.

matthieu-lapeyre replied:
maybe but which one ? ...

damiencaselli replied:
Ah my bad, I read the link was leading to this repository.

damiencaselli commented on _pages/about/en.md:
```diff
@@ -11,50 +11,43 @@ published: true
+## Usage rights
 
-### Web control interface
+All the technological development work made in the Poppy project is freely available under open source licenses. Only the name usage *"Poppy"* is restricted and protected as an international trademark to protect its identity.
```
Should we refer to « the Poppy project » or « Poppy Project »?

matthieu-lapeyre replied:
The poppy project like the visual programming :)

matthieu-lapeyre committed:
Copy "about" en content to fr

dashdashzako committed:
Add missing contact page

Contact form is sent to formspree.io, which then forwards it to
`site.data.social.email` and redirects to a "thank you" page.

dashdashzako committed:
HTML proofer must ignore flowers.inria.fr url
